worker/src/scripts/upsertDefaultModelPrices.ts

```diff
@@ -0,0 +1,192 @@
+import { z } from "zod";
```

Comment:
Would we need any locking to ensure that only one worker performs those updates at a time? Should be quite easy by setting a timestamp and id in redis on start and unlocking after completion. That way we may avoid that we perform the same update for the same version N times.

Reply:
Here I think the additional logic would not be worth the gain - most of the times we would not do any upserting at all due to the initial updated_at check and in case of upserting the upserts are cheap. Do you have a strong opinion here?